### PR TITLE
Update java plugin to 0.69.0

### PR DESCRIPTION
**Pull request by svor**
### What does this PR do?
Updates java11 plugin to 0.69.0
Migrate java plugin to Java 11

### Screenshot/screencast of this PR
~~Now [VsCode Java extension asks user to import projects](url) after opening the workspace it is important to start initialize the project.~~ Solved, `java.server.launchMode:Standard` preference was added into devfiles which use redhat/java11 plugin in https://github.com/eclipse/che-devfile-registry/pull/297. 
 
screenshot-che openshift io-2020 10 22-14_20_51

Testing updated plugin: 

screenshot-che openshift io-2020 10 22-14_02_04

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/18170

### How to test this PR?

It could be tested on che.openshift.io by using this devfile:
<details>
<summary>Spring Pet Clinic Devfile</summary>

```yaml
apiVersion: 1.0.0
metadata:
  name: java-web-spring
attributes:
  persistVolumes: 'false'
projects:
  - name: java-web-spring
    source:
      location: 'https://github.com/spring-projects/spring-petclinic.git'
      type: git
      branch: main
components:
  - memoryLimit: 1512Mi
    preferences:
      java.server.launchMode: Standard
    type: chePlugin
    reference: 'https://raw.githubusercontent.com/svor/che-plugin-registry/sv/java69/v3/plugins/redhat/java11/0.69.0/meta.yaml'
  - mountSources: true
    endpoints:
      - name: 8080-tcp
        port: 8080
    memoryLimit: 768Mi
    type: dockerimage
    volumes:
      - name: m2
        containerPath: /home/user/.m2
    image: 'quay.io/eclipse/che-java11-maven:7.20.0'
    alias: tools
    env:
      - value: $(JAVA_OPTS)
        name: MAVEN_OPTS
      - value: ''
        name: MAVEN_CONFIG
      - value: '-XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom -Duser.home=/home/user'
        name: JAVA_OPTS
commands:
  - name: maven build
    actions:
      - workdir: '${CHE_PROJECTS_ROOT}/java-web-spring'
        type: exec
        command: mvn clean install
        component: tools
  - name: run webapp
    actions:
      - workdir: '${CHE_PROJECTS_ROOT}/java-web-spring'
        type: exec
        command: |
          java -jar -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005 \
          target/*.jar
        component: tools
  - name: Debug remote java application
    actions:
      - referenceContent: |
          {
          "version": "0.2.0",
          "configurations": [
            {
              "type": "java",
              "name": "Debug (Attach) - Remote",
              "request": "attach",
              "hostName": "localhost",
              "port": 5005
            }]
          }
        type: vscode-launch
```

</details>

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by benoitf** [comment]
Should all java devfiles be updated to not have the vscode-java prompt ?

**Comment by svor** [comment]
@benoitf 
> Should all java devfiles be updated to not have the vscode-java prompt ?
> 
> to have `java.server.launchMode:Standard`

As for me it is a good idea and I think with this preference we don't need to change tests

File: v3/plugins/redhat/java11/0.69.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-java:11-7bd8c8c"
+      name: vscode-java
+      memoryLimit: "1500Mi"
+      volumes:
+      - mountPath: "/home/theia/.m2"
+        name: m2
+  extensions:
+    - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-java-debug/vscode-java-debug-0.26.0.vsix
+    - https://open-vsx.org/api/redhat/java/0.69.0/file/redhat.java-0.69.0.vsix
```
Comment (ericwill) [inline v3/plugins/redhat/java11/0.69.0/meta.yaml:23]: Let's use https://download.jboss.org/jbosstools/static/jdt.ls/stable/java-0.69.0-2547.vsix

File: v3/plugins/redhat/java/0.69.0/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-java:11-7bd8c8c"
+      name: vscode-java
+      memoryLimit: "1500Mi"
+      volumes:
+      - mountPath: "/home/theia/.m2"
+        name: m2
+  extensions:
+    - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-java-debug/vscode-java-debug-0.26.0.vsix
+    - https://open-vsx.org/api/redhat/java/0.69.0/file/redhat.java-0.69.0.vsix
```
Comment (ericwill) [inline v3/plugins/redhat/java/0.69.0/meta.yaml:23]: Let's use https://download.jboss.org/jbosstools/static/jdt.ls/stable/java-0.69.0-2547.vsix

**Review by ericwill — APPROVED** [review]

**Pull request merged**